analysis/bin/BuildFile.xml

```diff
@@ -68,3 +68,16 @@
   <use name="nano/analysis"/>
   <flags CXXFLAGS="-g" />
 </bin>
+
+<!--erase this line after executing lines in nano/analysis/test/hh/external_library_recipe.sh
```

Comment:
why erase line?

Reply:
because external libraries should be installed to build the analyser but not everybody needs to install them 
should I remove include lines in the analyser instead...?